<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LumaMixTransition</class>
 <widget class="QWidget" name="LumaMixTransition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Transition</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Video</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="lumaCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Dissolve</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cut</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bar Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bar Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barn Door Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barn Door Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barn Door Diagonal SW-NE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barn Door Diagonal NW-SE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diagonal Top Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diagonal Top Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Waterfall Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Waterfall Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Snake Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Snake Parallel Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Snake Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Matrix Snake Parallel Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barn V Up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iris Circle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double Iris</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iris Box</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box Bottom Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box Bottom Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box Right Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Clock Top</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom...</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="customLumaLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="invertCheckBox">
       <property name="toolTip">
        <string>Swap the appearance of the A and B clips</string>
       </property>
       <property name="text">
        <string>Invert Wipe</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="softnessLabel">
         <property name="text">
          <string>Softness</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="softnessSlider">
         <property name="toolTip">
          <string>Change the softness of the edge of the wipe</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="softnessSpinner">
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Audio</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="crossfadeRadioButton">
         <property name="toolTip">
          <string>Automatically fade-out the audio of clip A
and fade-in the audio of clip B over the
duration of the transition.</string>
         </property>
         <property name="text">
          <string>Cross-fade</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1,0,0">
       <item>
        <widget class="QRadioButton" name="mixRadioButton">
         <property name="toolTip">
          <string>Do not alter the audio levels during the
course of the transition. Instead, set a
fixed mixing level, or choose only clip A's
audio (0%) or clip B's audio (100%).</string>
         </property>
         <property name="text">
          <string>Mix:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="mixSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mixSpinner">
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lumaCombo</tabstop>
  <tabstop>invertCheckBox</tabstop>
  <tabstop>softnessSlider</tabstop>
  <tabstop>softnessSpinner</tabstop>
  <tabstop>crossfadeRadioButton</tabstop>
  <tabstop>mixSlider</tabstop>
  <tabstop>mixSpinner</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>softnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>softnessSpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>softnessSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>softnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixSpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
